client/src/components/common/Menu/index.js

```diff
@@ -43,5 +45,16 @@ const Menu = ({ showOrHide, isSigned: protect, show }) => {
     </Router>
   );
 };
+Menu.propTypes = {
+  showOrHide: PropTypes.func,
```

Comment:
this should probably be required (`PropTypes.func.isRequired`) - in which case you can remove it from default props

Reply:
Done